definitions/checks/foreman/check_external_db_evr_permissions.rb

```diff
@@ -0,0 +1,56 @@
+module Checks
+  module Foreman
+    class CheckExternalDbEvrPermissions < ForemanMaintain::Check
+      metadata do
+        label :external_db_evr_permissions
+        for_feature :foreman_database
+        description 'Check that external DBs have proper EVR extension permissions'
+        tags :pre_upgrade
+        confine do
+          feature(:foreman_database) && !feature(:foreman_database).local? && feature(:katello)
+        end
+      end
+
+      def run
+        return unless evr_exists?
+
+        error_msg = 'The evr extension is not owned by the foreman DB owner. Please run the ' \
```

Comment:
I think we should include mention of running it on the external database server just to be as clear as possible about where they would be executing the statement.

Reply:
The only failures reasons I can think of would be if the DB connection failed or if the config doesn't have the correct username information. I've updated the error message.